src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -665,7 +665,7 @@ static void fillQueryEntity(BSONArrayBuilder& ba, const EntityId* enP)
     ent.append(idString, enP->id);
   }
 
-  if (enP->type != "")
+  if ((enP->type != "") && (enP->type != "Thing"))  // FIXME PR: Is this OK (Thing)? (POST /v2/op/query needs it ...)
```

Comment:
By mysterious .DISABLED test I mean 1068_metadata_value_as_compound/metadata_value_as_compound_array.test.DISABLED, to be "recovered" just after this PR.